h2o-core/src/main/java/water/api/schemas3/ModelParametersSchemaV3.java

```diff
@@ -209,7 +209,6 @@ public P fillImpl(P impl) {
 
     impl._train = (this.training_frame == null) ? null : Key.<Frame>make(this.training_frame.name);
     impl._valid = (this.validation_frame == null) ? null : Key.<Frame>make(this.validation_frame.name);
-    impl._max_runtime_secs = nfolds > 0 ? max_runtime_secs / (nfolds+1) : max_runtime_secs;
```

Comment:
@michalkurka : for your concern, this is one of the main changes related with model building, (same in GridSearch): this time-repartition logic is now fully done in ModelBuilder.